src/Base/XMLTools.h

```diff
@@ -28,16 +28,10 @@
 #include <memory>
 #include <iostream>
 #include <xercesc/util/TransService.hpp>
+#include <xercesc/util/XercesDefs.hpp>
 
 #include <Base/Exception.h>
-
-
-XERCES_CPP_NAMESPACE_BEGIN
-    class DOMNode;
-    class DOMElement;
-    class DOMDocument;
-XERCES_CPP_NAMESPACE_END
-
+XERCES_CPP_NAMESPACE_USE

```

Comment:
Same here.

Reply:
Yes, my wrong, i got a little crazy trying to solve "DOMDocument redefinition error" at first.